your-first-cluster.md

```diff
@@ -51,6 +51,7 @@ export CLUSTERCTL_REPOSITORY_PATH=$(pwd)/clusterctl/repository
 
 # Set the CLUSTER_TOPOLOGY variable to true. This is a feature flag used to enable ClusterClass.
 export CLUSTER_TOPOLOGY=true
+export EXP_RUNTIME_SDK=true
```

Comment:
Just did this in another PR and did it on all operating systems + adjusted the comment above.

(So rebase should solve it)